### PR TITLE
Keep documentation up-to-date

### PR DESCRIPTION
**Pull request by macmiranda**
Hi,

Just wanted to make sure that the docs are using the latest tags.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,15 +98,15 @@ Following inputs can be used as `step.with` keys
...
 
 ## Example 4 - Separate AWS login
-```
-    - name: Configure AWS credentials     # using OIDC instead of IAM credentials
+```yaml
+    - name: Configure AWS credentials
       uses: aws-actions/configure-aws-credentials@v1
       with:
-        role-to-assume: arn:aws:iam::${{ fromJson(secrets.ACCOUNT_IDS)[env.environment] }}:role/github-actions
+        role-to-assume: arn:aws:iam::${{ env.aws-account-id }}:role/${{ env.aws-assume-role }}
```
Comment (mickmcgrath13) [inline README.md:105]: can you have dashes in env vars?

**Comment by mickmcgrath13** [comment]
one comment/question. otherwise, looks good!

File: README.md (hunk 1 above)
Comment (macmiranda) [reply README.md:105]: Yep

Comment (macmiranda) [reply README.md:105]: CAPS_NOTATION also possible

**Review by mickmcgrath13 — APPROVED** [review]

**Pull request merged**